pkg/apis/machine/v1alpha1/constants/types_constants.go

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package constants
+
+const (
+	// AnnotationKeyMachineUpdateFailedReason is the annotation key for the machine indicating the reason for the machine update failure.
```

Comment:
Please improve docstrings for all these constants. Target objects need to be clearly mentioned